.travis.yml

```diff
@@ -8,13 +8,11 @@ install:
   - pip install -r requirements.txt
```

Comment:
I think u can have all the installations for your modules in the requirements.txt like `coveralls`, `codecov`, `nose` with their specific versions and `pip install -r requirements.txt` can install all of them at once.

Reply:
> I think u can have all the installations for your modules in the requirements.txt like `coveralls`, `codecov`, `nose` with their specific versions and `pip install -r requirements.txt` can install all of them at once.

Oh, that's sound great let me try that. Thank for the enlightenment